otterdog/eclipse-apoapsis.jsonnet

```diff
@@ -72,7 +72,7 @@ orgs.newOrg('eclipse-apoapsis') {
           ],
           requires_pull_request: true,
           required_approving_review_count: 1,
-          required_status_checks: [
+          required_status_checks+: [
```

Comment:
https://gitlab.eclipse.org/eclipsefdn/it/api/git-eca-rest-api/-/issues/161 is not yet closed, is the ECA check already executed by the merge queue?

Reply:
I pinged the project about the status as I am also not aware tbh.

Reply:
I will also check when I see the next PR in our merge queue.

Reply:
Thanks, I also just confirmed that it is not yet executed in the merge queue.